<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkOrbitAnimationWidget</class>
 <widget class="QWidget" name="QmitkOrbitAnimationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkOrbitAnimationWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="angleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="buddy">
      <cstring>angleSlider</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="angleSlider">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="angleSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>Â°</string>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="reverseCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>angleSlider</tabstop>
  <tabstop>angleSpinBox</tabstop>
  <tabstop>reverseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
